itensor/mps/sweeps.h

```diff
@@ -498,6 +499,20 @@ sweepnext(int &b, int &ha, int N, Args const& args = Args::global(), int min_b =
         }
     }
 
+//void inline
+//sweepnext(int &b, int &ha, int N, int min_b = 1)
```

Comment:
For backwards compatibility, we should keep this version of `sweepnext`, but change it to `int min_b` instead of `int min_b = 1`. The point is to have backwards compatibility for someone who had called `sweepnext(1,1,20,2)` (which in this pull request right now would result in an error). If you call `sweepnext(1,1,20)` it would call the new version.